src/audio/Audio.js

```diff
@@ -228,12 +229,7 @@ Audio.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 		this.detune = value;
 
-		if ( this.source.detune === undefined ) {
-
-			console.warn( 'THREE.Audio: AudioBufferSourceNode.detune not supported by the browser.' );
```

Comment:
I have remove this log since the amount of warnings in Safari was just too high. We should keep in mind that `Audio.setDetune()` is called in `Audio.play()`.